Core/GDCore/Extensions/PlatformExtension.h

```diff
@@ -437,6 +476,21 @@ class GD_CORE_API PlatformExtension {
    * generator.
    */
   void StripUnimplementedInstructionsAndExpressions();
+
+  /**
+   * \brief Get an extension property
+   * \param propertyName The name of the property
+   */
+  gd::PropertyDescriptor& GetProperty(const gd::String& propertyName) {

```

Comment:
I don't find where this is used, can you show me?

Reply:
oh yes this is some dead code because I originally planned to implement it another way that wouldn't have worked. I'll remove that.